client/foundries.go

```diff
@@ -204,6 +204,28 @@ type TufCustom struct {
 	OverridesSha   string                `json:"meta-subscriber-overrides-sha,omitempty"`
 }
 
+// ota-tuf serializes root.json differently from Notary. The key representation
+// and signing algoritms differ slightly. These Ats* structs provide an
+// an implementation compatible with ota-tuf and libaktualizr.
+type AtsKeyVal struct {
+	Public  string `json:"public,omitempty"`
+	Private string `json:"private,omitempty"`
+}
+type AtsKey struct {
+	KeyType  string    `json:"keytype"`
+	KeyValue AtsKeyVal `json:"keyval"`
+}
+type AtsRootMeta struct {
+	tuf.SignedCommon
```

Comment:
Unfortunately, there is an `expires time.Time` field which Golang marshals in a format not understood by aktualizr.
Please, see how I overcome this in my waves PR and apply here too.

Reply:
This works because this line sets the time properly: https://github.com/foundriesio/fioctl/pull/83/files#diff-9b655247cb8326727517d0aeca1b07126f2602091675507157316aa1b4d1ac95R177

Reply:
Do you mean that if we round a time to a second (strip microseconds) - Golang will format this properly ?
I that's true - I will to use it as well in my PR.